msteams-platform/apps-in-teams-meetings/meeting-app-extensibility.md

```diff
@@ -200,7 +201,11 @@ The following list details the various user types along with their accessibility
     > [!NOTE]
     > Anonymous users inherit the global default user-level app permission policy. For more information, see [manage Apps](/microsoftteams/non-standard-users#anonymous-user-in-meetings-access).
 
-A guest or anonymous user can't add, remove, or uninstall apps.
+> [!NOTE]
+>
+> * If a guest user is a part of one-on-one or group chat, then apps cannot be added by anyone in the chat or meetings.
+>
+> * If federated users are added in a chat or call, then apps don't work for anyone.
```

Comment:
What happens to existing apps?

Reply:
Addressed in the docs

Reply:
Can you point to where this is explained? What is the behavior perceived by the app? Does that app know that this happened, or from the app's perspective does it look like users just stopped interacting with the app? How do we recommend apps handle this?

Reply:
1. @aosolis , @v-npaladugu , I recommend we make the following changes:
1. Move the notes section to below the table rather than add the top of the table
2. In Row 3, Column 2 of the table (Federated users <-> Tabs) update the line as Interaction is allowed in scheduled meetings
3. Remove the line "⁠A guest or anonymous user can't add, remove, or uninstall apps but they can interact with the apps added by others" since its already covered in the table
4. Add the following points to the notes section:

- 1. Federated users cannot interact with tab apps in calls or chat. 
2. If federated users are added to an existing call with in-tenant users, then all participants lose the ability to add, update or remove apps. However, In-tenant users can still interact with apps that were added before inviting federated users to the call.
3. Federated user behavior for apps in calls is consistent with the behavior experienced by federated users while using apps in group chats

Reply:
On second thoughts, w.r.t #3 above, rather than removing the line, we can change it to "but they can *partially interact with the apps ..."

Reply:
@v-npaladugu: Adding more comments based on Adrian's feedback:
1. Update the line, "The following table provides the user types and lists the features that each user can access" **to** "The following table provides the user types and lists the features that each user can access in scheduled meetings"
2. Add the following line before starting the bullet points in the notes section: "The behavior of all of the various user types for apps in calls is identical to their behavior in scheduled meetings with the exception of the following:" 
3. Change the 3rd point from "Federated user behavior for apps in Teams meetings and calls..." **to** "Federated user behavior for apps in calls is consistent with the behavior experienced by them while using apps in [group chats](https://docs.microsoft.com/en-us/microsoftteams/non-standard-users#usage-behavior-and-policy-for-federated-users)"